### PR TITLE
migration: add network documentation

### PR DESCRIPTION
**Pull request by jean-edouard**

### CODE REVIEW

File: docs/operations/live_migration.md [hunk 1]
```diff
@@ -151,7 +151,94 @@ These values can be change in the `kubevirt` CR:
...
+      "master": "eth1",
+      "mode": "bridge",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "10.1.1.0/24"
+      }
+    }'
+```
+
+### Configuring KubeVirt to migrate VM(I)s over that network
```
Comment (vladikr) [inline docs/operations/live_migration.md:197]: I think we should be consistent and keep referring to `VMI`.

**Comment by vladikr** [comment]
Thanks! 
/approve

File: docs/operations/live_migration.md [hunk 2]
```diff
@@ -151,7 +151,94 @@ These values can be change in the `kubevirt` CR:
...
+running `make cluster-up`:
+```
+export KUBEVIRT_NUM_NODES=2;
+export KUBEVIRT_NUM_SECONDARY_NICS=1;
+export KUBEVIRT_DEPLOY_ISTIO=true;
+export KUBEVIRT_WITH_CNAO=true
+```
+
+Here is a concrete example:
+https://asciinema.org/a/4Xszo18C1VzL8yBkyUaJfYSM7
```
Comment (cwilkers) [inline docs/operations/live_migration.md:239]: Unfortunately, this does not get converted to a clickable link by netlify. Could you mark it off in [] () instead?

**Review by cwilkers — CHANGES_REQUESTED** [review]
Everything looks good, I just caught one bare URL instead of a link.

File: docs/operations/live_migration.md (hunk 2 above)
Comment (jean-edouard) [reply docs/operations/live_migration.md]: Doh, the demo got archived... So I just removed the link, thanks for the review and sorry for the delay!

**Comment by jean-edouard** [comment]
@cwilkers could you please take another look? Thanks!

**Comment by cwilkers** [comment]
Thanks for the PR!

/approve
/lgtm

**Pull request merged**